### PR TITLE
Bulk add alerts to Timeline

### LINKED ISSUE #2695: [DOCS] Bulk alerts/events to Investigate in Timeline
## Description 

Related:
- https://github.com/elastic/kibana/pull/142737
- https://github.com/elastic/security-team/issues/3271

Add multiple events/alerts to investigate in Timeline.

## Docs to update

The main section to update is [Manage detection alerts | View alerts in Timeline](https://www.elastic.co/guide/en/security/master/alerts-ui-manage.html#signals-to-timelines). This section already describes two other ways of pivoting from an alert view to the Timeline view, and now we'll be adding a third way.

This could be very similar to [Change an alert's status](https://www.elastic.co/guide/en/security/master/alerts-ui-manage.html#detection-alert-status), which also explains three ways to do the same thing with alerts.

## Demo video

_(reproduced from https://github.com/elastic/kibana/pull/142737)_

https://user-images.githubusercontent.com/7485038/199056731-4287fc61-9d0b-4cf3-ba1f-741f6b66ae97.mov

### PR DESCRIPTION
**Pull request by benironside**
Fixes #2695 by updating the instructions for adding alerts to timeline to include new bulk add capabilities.

Preview: [View Alerts in Timeline](https://security-docs_2822.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html#signals-to-timelines)

### CODE REVIEW

File: docs/detections/alerts-ui-manage.asciidoc [hunk 1]
```diff
@@ -113,11 +113,16 @@ For information about exceptions and how to use them, refer to
 [[signals-to-timelines]]
 ==== View alerts in Timeline
 
-To view an alert in Timeline, click the *Investigate in timeline* button in the Alerts table. Alternatively, select *Take action* -> *Investigate in timeline* in the alert details flyout.
+* To view a single alert in Timeline, click the *Investigate in timeline* button in the Alerts table. Alternatively, select *Take action* -> *Investigate in timeline* in the alert details flyout.
```
Comment (joepeeples) [inline docs/detections/alerts-ui-manage.asciidoc:117]: Continuation of bullet, for left margin alignment:
```suggestion
+
```

File: docs/detections/alerts-ui-manage.asciidoc [hunk 2]
```diff
@@ -113,11 +113,16 @@ For information about exceptions and how to use them, refer to
...
 
 [role="screenshot"]
 image::images/timeline-button.png[Investigate in timeline button, 300]
 
+* To view multiple alerts in Timeline (up to 2,000), select them using the checkboxes on the left of the Alerts table, then click *Selected N alerts*, where *N* is the number of alerts selected, and click *Investigate in timeline*.
+
```
Comment (joepeeples) [inline docs/detections/alerts-ui-manage.asciidoc:122]: Continuation again:
```suggestion
+
```

File: docs/detections/alerts-ui-manage.asciidoc [hunk 3]
```diff
@@ -113,11 +113,16 @@ For information about exceptions and how to use them, refer to
 [[signals-to-timelines]]
 ==== View alerts in Timeline
 
-To view an alert in Timeline, click the *Investigate in timeline* button in the Alerts table. Alternatively, select *Take action* -> *Investigate in timeline* in the alert details flyout.
+* To view a single alert in Timeline, click the *Investigate in timeline* button in the Alerts table. Alternatively, select *Take action* -> *Investigate in timeline* in the alert details flyout.
 
 [role="screenshot"]
 image::images/timeline-button.png[Investigate in timeline button, 300]
 
+* To view multiple alerts in Timeline (up to 2,000), select them using the checkboxes on the left of the Alerts table, then click *Selected N alerts*, where *N* is the number of alerts selected, and click *Investigate in timeline*.
```
Comment (joepeeples) [inline docs/detections/alerts-ui-manage.asciidoc:121]: We probably don't need as much detail or explanation for a fairly common workflow, and we have a slightly different format for labels with variable text ([guideline here](https://wiki.elastic.co/pages/viewpage.action?pageId=105914797) in TL;DR section):

> Use _x_ (italicized) for variable text in UI component names, such as: "Click **Select all _x_ rules**, and then select **Activate**." 

```suggestion
* To view multiple alerts in Timeline (up to 2,000), select the checkboxes next to the alerts, then click *Selected _x_ alerts* -> *Investigate in timeline*.
```

**Review by joepeeples — COMMENTED** [review]
Added a couple of suggested edits, thanks!

**Review by jmikell821 — APPROVED** [review]
Merged feedback LGTM 👍

**Pull request merged**